web/client/utils/ConfigUtils.js

```diff
@@ -72,7 +72,7 @@ var ConfigUtils = {
         other.center = ConfigUtils.getCenter(other.center);
         return {
             map: other,
-            layers: layers.map(ConfigUtils.setBingKey),
+            layers: layers.map(ConfigUtils.setBingKey).map(ConfigUtils.setLayerId),
```

Comment:
Yes It'a mandatory and setLayerId is an helper function that set id equal to layer's name if id is missing